File: src/web/wcenzije/src/models/Review.ts
export interface Review {
  id: number;
  dateCreated: string;
  dateUpdated: string;
  imageUrls: string[];
  content: string;
  name: string;
  location: string;
  author: string;
  isAnonymous: boolean;
  likeCount: number;
  rating: number;
  qualities: Qualities;
  gender: Gender;
}

export interface Qualities {
  id: number;
  hasSoap: boolean;
  hasToiletPaper: boolean;
  hasPaperTowels: boolean;
  isClean: boolean;
}

export enum Gender {
  Unisex,
  Male,
  Female,
}

File: src/backend/Wcenzije/Wcenzije.Domain/Entities/Review.cs
using Wcenzije.Domain.Enums;

namespace Wcenzije.Domain.Entities
{
    public class Review
    {
        public long Id { get; set; }
        public DateTime DateCreated { get; set; }
        public DateTime DateUpdated { get; set; }
        public List<string> ImageUrls { get; set; }
        public string Content { get; set; }
        public string Name { get; set; }
        public string Location { get; set; }
        public string Author { get; set; }
        public bool IsAnonymous { get; set; }
        public int LikeCount { get; set; }
        public int Rating { get; set; }
        public Qualities Qualities { get; set; }
        public Gender Gender { get; set; }
    }
}
